lib/src/MuiPickersUtilsProvider.tsx

```diff
@@ -10,19 +10,20 @@ export interface MuiPickersUtilsProviderProps {
   children: React.ReactNode;
   locale?: any;
   libInstance?: any;
+  libFormat?: any;
```

Comment:
Its something I am planned to do! Thanks 👍 
But I think better naming will be something like `formats`, the same what we are passing down to adapter

Reply:
I wonder, have we considered this API?

```jsx
import React from "react"
import moment from "moment"
import "moment/locale/ko"
import MomentUtils from "@date-io/moment"
import { MuiPickersUtilsProvider, DatePicker } from "@material-ui/pickers"

moment.locale("ko")

const utils = new MomentUtils({
  locale: 'ko',
  format: {
    normalDate : "MMM Do, dddd",
  },
});

const CustomFormattedCalendar = () => {
  return (
    <MuiPickersUtilsProvider utils={utils}>
```